corehq/apps/geospatial/models.py

```diff
@@ -87,6 +96,11 @@ def _clear_caches(self):
     def max_travel_time_seconds(self):
         return self.max_case_travel_time * 60
 
+    @property
+    def mapbox_travel_mode(self):
+        assert self.travel_mode in [self.WALKING, self.CYCLING, self.DRIVING]
```

Comment:
Instead of doing an assert check here when the data is already saved, should we rather raise the assert when saving an instance that has an incorrect choice?

Reply:
Good idea!

Reply:
https://github.com/dimagi/commcare-hq/pull/34695/commits/0a86053ed31fa6ea5fa26d5b1e4111295148ad5d

Reply:
Thanks, looks good!